### PR TITLE
Add favourites ability , some fix

### PR DESCRIPTION
**Pull request by VolodymyrButs**

### CODE REVIEW

File: src/components/AllGames/AllGames.jsx [hunk 1]
```diff
@@ -45,42 +48,57 @@ function AllGames() {
...
+                            {topGames.map(item => {
+                                return (
+                                    <GamesCardSlider
+                                        key={item.id}
+                                        id={item.id}
+                                        title={item.displayName}
+                                        image={item.image}
+                                    />
+                                )
+                            })}
```
Comment (STDio96) [inline src/components/AllGames/AllGames.jsx:73]: you can remove `return` here:
`{topGames.map(item => (
    <GamesCardSlider
        key={item.id}
        id={item.id}
        title={item.displayName}
        image={item.image}
    />
))}`

File: src/components/AllGames/AllGames.jsx [hunk 2]
```diff
@@ -45,42 +48,57 @@ function AllGames() {
...
-                                />
-                            </div>
-                        )
-                    })}
-                </>
+                    {tagsWithoutTop &&
+                        tagsWithoutTop.map(tag => {
+                            return (
+                                <div
+                                    key={tag.displayName}
```
Comment (STDio96) [inline src/components/AllGames/AllGames.jsx:80]: `displayName` isn't always unique, so it's not recommended to use it as a `key`

File: src/components/AllGames/GamesContainer/GamesContainer.jsx [hunk 3]
```diff
@@ -4,43 +4,50 @@ import PropTypes from 'prop-types'
...
+                    </div>
                 </div>
             ) : (
-                <div className={styles['nothing-found']}>
-                    <span className={styles['message']}>
-                        {messages.nothingFound}
-                    </span>
-                    <PMSorry className={styles['image']} />
-                </div>
+                search && (
```
Comment (STDio96) [inline src/components/AllGames/GamesContainer/GamesContainer.jsx:41]: why do we use `search`? If we don't have anything there, we can just show the 'nothing found' message

File: src/components/GamePage/GamePage.jsx [hunk 4]
```diff
@@ -4,18 +4,18 @@ import { useSelector } from 'react-redux'
 
 import styles from './GamePage.module.css'
 
-import Header from '../Header/Header'
+import HeaderGame from '../Header/HeaderGame'
```
Comment (STDio96) [inline src/components/GamePage/GamePage.jsx:7]: why did you create another `HeaderGame`? Do we have a header only for the GameComponent?

File: src/redux/actions/constants.js [hunk 5]
```diff
@@ -5,3 +5,4 @@ export const SET_LOADED = 'data/set_loaded'
 export const SET_FILTERS_VISIBILITY = 'filters/set_filters_visibility'
 export const TOGGLE_PROVIDER = 'filters/toggle_provider'
 export const SET_CATEGORY = 'filters/set_category'
+export const SET_FAVOURITES = 'set_favourites'
```
Comment (STDio96) [inline src/redux/actions/constants.js:8]: `export const SET_FAVOURITES = 'data/set_favourites'` - just to make everything in 1 style

File: src/redux/actions/data.js [hunk 6]
```diff
@@ -25,3 +33,12 @@ export const fetchData = token => dispatch => {
             dispatch(setData(body))
         })
 }
+
+export const fetchFavourites = token => dispatch => {
+    fetchGetFavourite(token)
+        .then(data => data.json())
+        .then(body => {
+            dispatch(setFavourites(body))
+        })
```
Comment (STDio96) [inline src/redux/actions/data.js:42]: `if (response.ok) {...}`?

File: src/redux/reducers/data.js [hunk 7]
```diff
@@ -7,6 +12,7 @@ const initialState = {
         tags: [],
         games: [],
     },
+    favourites: [],
```
Comment (STDio96) [inline src/redux/reducers/data.js:15]: mb move them to `data { ... }`?

**Review by dkupchyk — APPROVED** [review]

**Pull request merged**